src/com/google/common/css/compiler/gssfunctions/GssFunctions.java

```diff
@@ -370,7 +375,7 @@ public String getCallResultString(List<String> args)
           baseColorString, args.get(1), args.get(2), args.get(3));
     }
 
-    protected String addHsbToCssColor(
+    public static String addHsbToCssColor(
```

Comment:
Ahh, I see that you're re-using this method down on line 503. Hmm. Yes, I would agree that you wouldn't want to have an instance of `AddHsbToCssColor` to color for that. I'm going to have to think a bit on this one.